src/Configuration/FileLocations.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Configuration;
+
+use Tightenco\Collect\Support\Collection;
+
+final class FileLocations
```

Comment:
why final?

Reply:
It was `final` already, just kept it.. I've read it's good practice to do so, and I think i agree: https://matthiasnoback.nl/2018/09/final-classes-by-default-why/

Reply:
There is a fixer in ECS that makes all classes final and there is a good reason not to use it: you can't mock final classes in tests.

Reply:
Also you block user to extend the class, why? Final classes shouldn't be used without interfaces: https://ocramius.github.io/blog/when-to-declare-classes-final/

Reply:
Removed. :-)